croc.service

```diff
@@ -42,8 +42,14 @@ RestrictSUIDSGID=true
 # Other bits are both set and locked.
 SecureBits=keepcaps-locked,noroot,noroot-locked,no-suid-fixup,no-suid-fixup-locked
 SystemCallArchitectures=native
-# List of permitted syscalls; any others will result in termination via SIGSYS.
+# List of permitted syscalls
 SystemCallFilter=@network-io @system-service
```

Comment:
`@network-io` is already within `@system-service` according to `systemd-analyze syscall-filter @system-service`

Reply:
I included the note regarding SIGSYS in case anyone unfamiliar with syscall filtering wonders why the program may be terminating with this signal.